docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+        * Click **Save**
+
+Add _Directory reader_ permission for the user to access the service principal
+* Select [Azure Active Directory](https://aad.portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview) from Azure portal
+* Select **Roles and administrators** from left navigation
+* Select **Directory readers** role
+* Click **Add assignments**
+    * Search username and click **Add**
+   
+Create service principal for AKS cluster using the following steps
+
+* Manually create service principal using following command
+    ```console
+    $ az ad sp create-for-rbac --skip-assignment --name myAKSClusterServicePrincipal
+    ```
+## Create resource group
+
+Create a resource group to place the cloud resources of your Scalar DL environment. You can use the following steps to create a resource group
+
+* Select [Resource groups](https://portal.azure.com/#blade/HubsExtension/BrowseResourceGroups) from Azure Portal.
+* Click **Create**
+* On the Basics page, configure the following options
+    * Select your **Subscription**
+    * Enter **Resource group** name
+    * Select **Region**
+    * Click **Review+Create**
+    * Click **Create**
+    
+## Create virtual network
+
+Create a virtual network based on the [requirements](./ManualDeploymentGuideScalarDLOnAzure.md#step-1-configure-your-network) for your Scalar DL environment using the following steps
+
+* Select [Virtual networks](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Network%2FvirtualNetworks) from Azure portal
+* Click **Create**
+* On the Basics page, configure the following options
+    * Select your **Subscription**
+    * Select the **Resource group**
+    * Enter **Name**
+    * Select **Region**
+    * Click **Next: IP Addresses**
+* On the IP Addresses page, enter the following values for the virtual network.
+    * Enter **IPv4 Address space** as _10.42.0.0/16_ for virtual network.
+    * Click **Add subnet** and add the following subnets
+        * Enter **Subnet name** as _public_ and **subnet address range** as _10.42.0.0/24_ then click **Add**
+        * Enter **Subnet name** as _k8s_node_pod_ and **subnet address range** as _10.42.40.0/22_ then click **Add**
+        * Enter **Subnet name** as _k8s_ingress_ and **subnet address range** as _10.42.44.0/22_ then click **Add**
+    * Click **Review+create**
+    * Click **Create**
+
+## Create Cosmos DB
+
+You can use following steps to create Cosmos DB account based on the basis of [requirements](./SetupAzureDatabase.md#cosmos-db)
+
+* Select [Cosmos DB](https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.DocumentDb%2FdatabaseAccounts) from Azure portal
+* Click **Create**
+* Click **Create** in _Core (SQL) - Recommended_
+* Select your **Subscription**
+* Select your **Resource group**
+* Enter **Account Name**
+* Select **Location**
+* Select **Capacity mode** as _Provisioned throughput_
```

Comment:
I think we need to customize the networking.
The default settings allow access from all networks, I think we should only allow access from the `k8s_node_pod` subnet.

Reply:
I didn't add it since this document is only an example guide.